docs/lang-ref-ytt.md

```diff
@@ -100,6 +100,66 @@ md5.sum("data") # "8d777f385d3dfec8815d20f7496026dc"
 sha256.sum("data") # "3a6eb0790f39ac87c94f3856b2dd2c5d110e6811602261a9a923d3bb23adc8b7"
 ```
 
+#### Library Loading
```

Comment:
let's separate this into dedicated file: `See [Library loading specific docs](lang-ref-ytt-lib.md).`